portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
```

Comment:
Fwiw you don't have to include this copyright disclaimer in your own files. Leaving it in is fine too, it's up to you.